src/networkcloud/azext_networkcloud/operations/virtualmachine/_create.py

```diff
@@ -2,65 +2,47 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-# pylint: disable=wildcard-import,unused-wildcard-import,too-many-nested-blocks
-# pylint: disable=protected-access, too-few-public-methods,line-too-long
-# flake8: noqa
+
+# pylint: disable=too-many-nested-blocks
+# pylint: disable=protected-access,too-few-public-methods
 
 """
 Provides VirtualMachineCreate customization
 """
 
-import os
-
-from azure.cli.core import keys
-from azure.cli.core.aaz import *
+from azure.cli.core.aaz import register_callback
 from knack.log import get_logger
-from azure.cli.core.azclierror import InvalidArgumentValueError
-
-from ...aaz.latest.networkcloud.virtualmachine import \
-    Create as _VirtualMachineCreate
+from azext_networkcloud.aaz.latest.networkcloud.virtualmachine import Create as _Create
+from ..common_ssh import CustomSshOptions
 
 logger = get_logger(__name__)
 
-class VirtualMachineCreate(_VirtualMachineCreate):
-    '''
+
+class Create(_Create, CustomSshOptions):
+
+    """
     This custom code inherits from generate virtual machine functions. It is
     integrated into the generated code via:
-    cli-ext/v20221212preview/ext/src/networkcloud/azext_networkcloud/commands.py
-    '''
+    cli-ext/v*/ext/src/networkcloud/azext_networkcloud/commands.py
+    """
 
     @classmethod
     def _build_arguments_schema(cls, *args, **kwargs):
         args_schema = super()._build_arguments_schema(*args, **kwargs)
 
         # Build VM Authentication args
-        args_schema.generate_ssh_keys = AAZBoolArg(
-            options=["--generate-ssh-keys"],
-            arg_group="Authentication",
-            help="Generate SSH public and private key files if missing. The keys will " +
-            "be stored under '~/.ssh'."
+        args_schema = CustomSshOptions.build_ssh_arg_schema(
```

Comment:
You can use `cls` as it inherited the `CustomSshOptions` class
```suggestion
        args_schema = cls.build_ssh_arg_schema(
```

Reply:
Updated